extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -149,6 +149,19 @@ private function getRelatedPages( $articleId, $limit = 6 ){
 		}
 	}
 
+	private function getCuratedContentSections() {
+		try {
+			$response = $this->sendRequest( 'CuratedContent', 'getList' )->getData();
+			if ( !empty( $response['sections'] ) ) {
+				return $response['sections'];
```

Comment:
What about `featured` field? According to spec we should display its items on the top of the main page.

Featured section isn't listed in `sections` so we would lose this information. For example http://gameofthrones.wikia.com/wikia.php?controller=CuratedContent&method=getList